Description: Disable "vendored" feature
 This feature relies on vendored library C code, which we want to exclude from
 Debian packages.
Author: Arnaud Ferraris <aferraris@debian.org>
Forwarded: not-needed
Last-Update: 2023-11-22
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: a/Cargo.toml
===================================================================
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -54,9 +54,6 @@
 [build-dependencies.pkg-config]
 version = "0.3"
 
-[features]
-vendored = []
-
 [target."cfg(target_env = \"msvc\")".build-dependencies.vcpkg]
 version = "0.2"
 
